Userland/Applications/GamesSettings/CardSettingsWidget.gml

```diff
@@ -42,7 +42,6 @@
 
             @GUI::ComboBox {
                 name: "cards_front_image_set"
-                model_only: true
```

Comment:
Is there any particular reason why we are removing these instead of making them accessible to the GMLCompiler? It _seems_ like we should keep their effect if possible.

Reply:
Hmm, you are right. As discussed on discord - it seems like this property has a name mismatch, I've found the correct method that can set this property and it's `set_only_allow_values_from_model` from ComboBox widget, so now I've changed property's name to `only_allow_values_from_model` instead of `model_only`, and it seems to be working.